source/unified-test-format/unified-test-format.rst

```diff
@@ -1239,6 +1239,7 @@ expectedCmapEvent
   The structure of this object is as follows:
 
   - ``hasServiceId``: Defined in `hasServiceId`_.
+  - ``closeInUseConnections``: Defined in `closeInUseConnections`_.
```

Comment:
Unlike `hasServiceId`, `closeInUseConnections` is only relevant to `poolClearedEvent`. I see no reason to refer to a common definition below. I suggest rewriting as follows:

```suggestion
  - ``closeInUseConnections``: Optional boolean. If specified, test runners MUST assert that the field is set and matches this value.
```